pic32/cores/pic32/Tone.cpp

```diff
@@ -145,6 +145,9 @@ uint8_t port;
 //************************************************************************
 void disableTimer(uint8_t _timer)
 {
+    // Added to prevent compiler warning about unused paraemer
```

Comment:
There's a slightly better way to do this with GCC - add __attribute__ ((unused)) right after the parameter. Not important enough to fix, but interesting.